certified-connectors/Tribal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2419 @@
+            "x-ms-dynamic-values": {
+              "operationId": "ActionEntities",
+              "value-path": "name",
+              "value-title": "description",
+              "parameters": {
+                "actionType": "Create"
+              }
+            },
+            "x-ms-url-encoding": "single",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "x-correlation-id",
+            "x-ms-summary": "Correlation Id",
+            "description": "Correlation Id added in the x-correlation-id header",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "x-ms-dynamic-schema": {
+                "operationId": "ActionSchema",
+                "parameters": {
+                  "actionType": "Create",
+                  "entity": {
+                    "parameter": "entityType"
+                  },
+                  "schemaType": "Body"
+                },
+                "value-path": "properties"
+              },
+              "x-ms-dynamic-properties": {
+                "operationId": "ActionSchema",
+                "parameters": {
+                  "actionType": {
+                    "value": "Create"
+                  },
+                  "entity": {
+                    "parameterReference": "entityType"
+                  },
+                  "schemaType": {
+                    "value": "Body"
+                  }
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "type": "object",
+              "x-ms-dynamic-schema": {
+                "operationId": "ActionSchema",
+                "parameters": {
+                  "entity": {
+                    "parameter": "entityType"
+                  },
+                  "actionType": "Create",
+                  "schemaType": "Response"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "403": {
+            "description": "Forbidden"
+          }
+        }
+      },
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "x-ms-dynamic-schema": {
+                "operationId": "ActionSchema",
+                "parameters": {
+                  "entity": {
+                    "parameter": "entityType"
+                  },
+                  "actionType": "ReadCollection",
+                  "parentEntity": "",
```

Comment:
@mszy-tribal Empty property values in the **swagger** can be removed until unless you have a specific reason to keep them.

Reply:
they have been removed